commit 42915074c2bd19759625754b04f3a34a3b140037
Author: Maxwell Pray <synthead@gmail.com>
Date:   Wed Oct 29 17:16:48 2014 -0700

    Replace references to "C-" and "S-" with "Ctrl-" and "Shift-" for clarity.
    Remove text to explain what "C-" and "S-" means.

diff --git a/xwax.1 b/xwax.1
index 73b38db..87286ec 100644
--- a/xwax.1
+++ b/xwax.1
@@ -185,36 +185,32 @@ controlled via the incoming timecode signal from the turntables.
 The keyboard provides additional controls.
 
 .P
-"C-" and "S-" means a keypress is combined with
-the 'Control' or 'Shift' key, respectively.
-
-.P
 Record selection controls:
 
 .TP
-cursor up, cursor down
+Up, Down
 Move highlighted record up/down by one.
 
 .TP
-page up, page down
+Page Up, Page Down
 Scroll the record listing up/down by one page.
 
 .TP
-left cursor, right cursor
+Left, Right
 Switch to the previous/next crate of records.
 
 .TP
-tab
+Tab
 Toggle between the current crate and the 'All records' crate.
 
 .TP
-C-tab
+Ctrl-Tab
 Toggle sort mode between: artist/track name, BPM and 'playlist'
 order. Playlist order is the order in which records were returned
 from the scanner.
 
 .TP
-C-S-tab
+Ctrl-Shift-Tab
 Re-scan the currently selected crate.
 
 .P
@@ -231,7 +227,7 @@ Deck 0	Deck 1	Deck 2
 F1	F5	F9	Load currently selected track to this deck
 F2	F6	F10	Reset start of track to the current position
 F3	F7	F11	Toggle timecode control on/off
-C-F3	C-F7	C-F11	Cycle between available timecodes
+Ctrl-F3	Ctrl-F7	Ctrl-F11	Cycle between available timecodes
 .TE
 
 .P
@@ -242,7 +238,7 @@ flag on the command line.
 Audio display controls:
 
 .TP
-C-+, C--
+Ctrl-+, Ctrl--
 Zoom in/out the close-up audio meters for all decks.
 
 .SH NOVATION DICER CONTROLS